### DISCUSSION TITLE
Saving model after Hotword Adjusting

### DISCUSSION THREAD

**Discussion by mrinal-sourav** [coqui-ai/STT-examples#32] [General]
Hi Team, 

So, I am trying to adjust hotwords to increase Coqui model accuracy on Speech Commands dataset. Without any adjustment I get 55% accuracy on the entire dataset (considered as classification). 

I copied the hotwords_adjusting.py script as is in the training folder and run the following command in the docker container:
python -m coqui_stt_training.hotword_adjusting --model /data/stt_v1.0.0-large-vocab/model.tflite --scorer /data/stt_v1.0.0-large-vocab/large_vocabulary.scorer --audio /data/bird/1a4259c3_nohash_0.wav --min -20.0 --max 20.0 --steps 3 --hot_words bird 

I get the following output:

TensorFlow: v2.8.0-8-g06c8fea58fd
 Coqui STT: v1.3.0-0-g148fa743
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
['bird'] = (-20.0,) :: [for]
['bird'] = (0.0,) :: [for]
['bird'] = (20.0,) :: [bird ]

Does this mean that the the model would now predict "bird" instead of "for" given the same audio?

Also, how do l save the model after the boosting?

**Comment by reuben** [comment]
@JRMeyer
